packages/lit-dev-content/src/components/litdev-playground-share-gist.ts

```diff
@@ -91,7 +91,7 @@ export class LitDevPlaygroundShareGist extends LitElement {
   }
 
   private get _signInButton() {
-    return html`<litdev-icon-button @click=${this._signIn}>
+    return html`<litdev-icon-button id="signInButton" @click=${this._signIn}>
```

Comment:
No action required - can resolve this.
This is a really silly comment (I am sorry), but does prettier format this html?
Asking because the `saveNewGistButton` beneath has been formatted differently.

Reply:
Yep, these are prettier formatted. It did a single line here because it's just short enough to fit, whereas the next one was too long so it broke up the lines.